subsys/usb/usb_c/usbc_pe_common.h

```diff
@@ -22,6 +28,102 @@ enum pe_error {
 	ERR_XMIT,
 };
 
+/**
+ * @brief Policy Engine Layer States
+ */
+enum usbc_pe_state {
```

Comment:
Up to this point, things like state IDs and flags were contained in the state machine .c files, and that meant that only that state machine could access them by definition. Now files from other state machines are including this one, and that might make someone think that it's OK to touch this stuff from outside the PE. I think it might be safer to have something like `usbc_pe_*_internal.h` for things that need to be shared among PE files but are not part of the PE's public API.

Same concern for the TC.

Reply:
Done

Reply:
I guess there are multiple levels of internal. Except for `usbc_stack.h`, all of this stuff is internal to the PD stack, but the things I was talking about are further internal to the PE or the TC, and this change elevates their visibility to broadly within the PD stack.

What I had in mind here is that the public API of the PE would remain in something like `usbc_pe.h`, and files in the TC or the PRL or the USB-C task would include `usbc_pe.h`. There would be a separate file like `usbc_pe_common_internal.h`, and only other files in the PE, like `usbc_pe_common.c` or `usbc_pe_sink.c`, would include `usbc_pe_common_internal.h`. I wasn't thinking of changing the name of the sink states header file to include internal as you just did, but I like it.

Same deal with the TC.

Reply:
I've updated the code. Thanks